app/test_main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
+
+
+@pytest.fixture(scope="module")
+def url() -> str:
+    return "https://www.google.com"
+
+
+@pytest.mark.parametrize("internet_connection,"
+                         "valid_url,"
+                         "expected_output",
+                         [
+                             pytest.param(True, True, "Accessible",
+                                          id="internet_available_success"),
+                             pytest.param(False, True, "Not accessible",
+                                          id="internet_unavailable_success"),
+                             pytest.param(True, False, "Not accessible",
+                                          id="internet_available_fail"),
+                             pytest.param(False, False, "Not accessible",
+                                          id="internet_unavailable_fail")
+                         ])
```

Comment:
why do u have `[]` in one style and `()` in completely different one?

Reply:
?

Reply:
> why do u have `[]` in one style and `()` in completely different one?

@pytest.mark.parametrize("test_input,expected", [("3+5", 8), ("2+4", 6), ("6*9", 42)])
in offical documentation uses same style

Reply:
It is not about documentation. It is about PEP8. Your `[]` and `()` are breaking lines differently

Reply:
I hope I understood correctly